components/search_engine_choice_strings.grdp

```diff
@@ -73,5 +72,11 @@
     Change
   </message>
 
-</if>
+  <message name="IDS_SEARCH_ENGINE_FALLBACK_MARKETING_SNIPPET" desc="This string describes what a search engine can do. The string is only shown if a search engine does not provide their own marketing snippet. The string is ambiguous and has a neutral, unopinionated tone.">
+    You can use <ph name="SEARCH_ENGINE_NAME">$1<ex>Brave</ex></ph> to search the web.
```

Comment:
Should this be "You can use Brave Search to search the web" or just "Brave"?

Reply:
This is an upstream string, we just replaced `Google` with `Brave` here, but the name here is just a placeholder. In https://source.chromium.org/chromium/chromium/src/+/main:components/search_engines/prepopulated_engines.json;l=77 the name is correct, so I assume that's what is used.